src/screens/dashboard/dashboard.tsx

```diff
@@ -47,6 +48,9 @@ const ANIMATED_BC_SELECTION = true;
 const calculateBalances = (accounts: IAccountState[]) => {
     return accounts.reduce(
         (out: any, account: IAccountState) => {
+            if (!account) {
```

Comment:
why do we need this check ?

Reply:
if somehow (from some error?) a null value gets in accounts state, app wont load anymore. basically this should never happen, we could remove it

Reply:
ok, i understand :)